substrate/bin/utils/chain-spec-builder/src/lib.rs

```diff
@@ -83,6 +83,10 @@ pub struct CreateCmd {
 	/// errors will be reported.
 	#[arg(long, short = 'v')]
 	verify: bool,
+	/// Chain properties in comma-separated KEY=VALUE format.
+	/// Example: `--properties tokenSymbol=UNIT,tokenDecimals=12,ss58Format=42,isEthereum=false`
```

Comment:
```suggestion
	/// Chain properties in `KEY=VALUE` format.
	///
	/// Multiple `KEY=VALUE` entries can be specified and separated by a comma.
	///
	/// Example: `--properties tokenSymbol=UNIT,tokenDecimals=12,ss58Format=42,isEthereum=false`
	/// Or: `--properties tokenSymbol=UNIT --properties tokenDecimals=12 --properties ss58Format=42 --properties=isEthereum=false`
	///
	/// The first uses comma as separation and the second passes the argument multiple times. Both
	/// styles can also be mixed.
```

Reply:
Added in [1b8e84a](https://github.com/paritytech/polkadot-sdk/pull/7368/commits/1b8e84a492dd696e0ca9cb9b29296bd5b8695758).